### DISCUSSION TITLE
Terrain Topology Algorithms

### DISCUSSION THREAD

**Discussion by felixcantet** [alelievr/Mixture#23] [Ideas]
Hey,

I'm currently working on a GPU implementation of this repository. It's a compilation of usefull algorithms for terrain texturing. 
I think it could be a good idea to add this algorithms in Mixture for generating terrain texture.

https://github.com/Scrawk/Terrain-Topology-Algorithms

What do you think about this ? 

Félix Cantet

**Comment by alelievr** [comment]
Hello, sorry I didn't see this message before :(

Yeah, it's definitely a good idea to add nodes for such operations, they all seem pretty straightforward (except maybe flow map) so they should be available for realtime mixtures.

**Reply by quatbol** [reply]
It would be amazing to have these in mixture.

**Reply by felixcantet** [reply]
Thank you for your feedbacks ! I will start working on it next week.

**Reply by alelievr** [reply]
Don't hesitate to ask if you have any question related to shader nodes / compute shader nodes :)

**Reply by felixcantet** [reply]
Thanks !

Is there any place where I can talk with you in french to be more comfortable if I have some specific questions?

**Reply by alelievr** [reply]
Hum, discord is fine if you have an account :)
